react-native-git-upgrade/checks.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+import {execSync} from 'child_process';
+import semver from 'semver';
+
+export function checkDeclaredVersion({declaredVersion}) {
+  if (!declaredVersion) {
+    throw new Error(
+      'Your \'package.json\' file doesn\'t seem to have \'react-native\' as a dependency.'
```

Comment:
Nit: Would it make sense to use double quotes so there's no need to escape the single quotes in many places?